Restor/Model/ImagingSession.h

```diff
@@ -32,6 +32,7 @@
 typedef NS_ENUM(NSInteger, ImagingStage) {
   ImagingStageNotStarted,
   ImagingStageImaging,
+  ImagingStageInverting,
```

Comment:
Does this last long enough to need appearing in the UI? And does it happen after the imaging stage?

Reply:
Yes, it lasts for several seconds taking the place of the "verifying" stage. In this case though asr does not report progress.